.azure-pipelines/ux-portal.yml

```diff
@@ -0,0 +1,46 @@
+# Variable 'BotAccessToken' was defined in the Variables tab
+# Multi-job configuration must be converted to matrix strategy: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration
+parameters: 
+- name: AzurePowerShellVersion
+  displayName: Azure PowerShell Version
+  type: string
+- name: IsLatestVersion
+  displayName: IsLatestVersion
+  type: boolean
+  default: true
+
+jobs:
+- job: Job_1
+  displayName: "Sync latest UX metadata files to Storage Blob"
+  timeoutInMinutes: 90
+  pool: pool-windows-2019
+  steps:
+  - checkout: self
+  - task: PowerShell@2
+    displayName: Compress the UX metadata files
+    inputs:
+      targetType: inline
+      script: >-
+        .\CompressUXMetadata.ps1
+      workingDirectory: tools
+  - task: AzurePowerShell@5
+    displayName: Upload to Storage Blob
+    inputs:
+      azureSubscription: 'azureps-infra-sp'
+      ScriptType: InlineScript
+      workingDirectory: artifacts
+      Inline: |
+        $context = New-AzStorageContext -StorageAccountName "$(StorageAccountName)"
+        If ("${{ parameters.IsLatestVersion }}" -eq "True")
+        {
+          Copy-Item "UX.zip" -Destination "azps-latest.zip"
+          Set-AzStorageBlobContent -Container $(StorageBlobContainerName) -Context $context -File "azps-latest.zip" -Blob "$(StorageBlobFolder)/azps-latest.zip" -Force
+        }
+        $zipFile = "${{ parameters.AzurePowerShellVersion }}.zip"
```

Comment:
What's the format of AzurePowerShellVersion? Is it "9.2.0" or "Az9.2.0"? How can we help who triggers this pipeline know this?

Reply:
Added a description on name and will also add step in release document.